### DISCUSSION TITLE
ImageMagick lite version?

### DISCUSSION THREAD

**Discussion by KFoster2** [ImageMagick/ImageMagick#2094] [Help]
I wonder if there is a lite version ImageMagick which takes up little space on my hard drive.

If not, is it possible to not install or remove the IMDisplay feature in order to reduce the size of installation? Or maybe downloading the source from a repository and build it on my own?

Thanks in advance!

**Comment by snibgo** [comment]
If you have a system for building C programs, you can build IM yourself. I show how I do this with Cygwin tools under Windows in [Compiling IM with Cygwin](http://im.snibgo.com/compim.htm).
